### PR TITLE
fix logout of cookie

### PR DESCRIPTION
**Pull request by davidmeirlevy**

### CODE REVIEW

File: server/controllers/logout.js [hunk 1]
```diff
@@ -12,21 +12,13 @@ async function logout(req, res, next) {
...
-			errors: {
-			  token: 'FAILED_TO_DELETE_TOKEN'
-			}
-		  })
-	} else {
-		if (authType === 'cookie') {
-			res = setCookie(res, '', -1);
-		}
-		res.status(200).end();
+	if (authType === 'cookie') {
```
Comment (itays123) [inline server/controllers/logout.js:15]: There is only one if block where the authType is 'cookie'. why don't you set the cookie there?

Comment (davidmeirlevy) [reply server/controllers/logout.js]: fixed 👍

**Comment by itays123** [comment]
Looks awesome!

**Review by drorrose — COMMENTED** [review]
Looks great

**Pull request merged**